fhir-persistence-scout/src/main/java/com/ibm/fhir/persistence/scout/cql/InsertParameterBlock.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.scout.cql;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.ibm.fhir.persistence.scout.SearchParameters.ParameterBlock;
+
+/**
+ * CQL command to insert the parameter block using the resource logical id as the key
+ */
+public class InsertParameterBlock {
+    
+    private final ParameterBlock parameterBlock;
+    private final byte[] payload;
+    
+    public InsertParameterBlock(ParameterBlock pb, byte[] payload) {
+        this.parameterBlock = pb;
+        this.payload = payload;
+    }
+
+    /**
+     * Run the command against the given session
+     * @param s
+     */
+    public void run(CqlSession s) {
+        
```

Comment:
Update in the code for the future? marking just to confirm

Reply:
Yes, this is a work in progress, but sharing to give others visibility.